.github/workflows/shfmt.yml

```diff
@@ -0,0 +1,25 @@
+name: Shfmt
+
+# shfmt formats shell programs
+# https://github.com/mvdan/sh#shfmt
+
+permissions:
+  contents: read
+
+on: [workflow_dispatch, pull_request, push]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    name: Shfmt - Lint
+    runs-on: ubuntu-20.04
+    container:
+      image: ghcr.io/build-trust/ockam-builder@sha256:e43dd94652096b03cc472a3c709c7335e8b166cab77b7a7b56f88fa38f3d24cc
+    steps:
+      - name: Check out ockam repository
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+      - name: Run Shfmt on all ''.sh' files
```

Comment:
Not sure if the quote format was intended

Reply:
That was unintended and it's fixed now.